listen.go

```diff
@@ -33,7 +32,7 @@ func (l *RelayListener) Accept() (manet.Conn, error) {
 		}
 
 		// TODO: Pretty print.
-		log.Infof("accepted relay connection: %s", c)
+		log.Infof("accepted relay connection: %q", c)
```

Comment:
is this the right format specifier?
godoc says:
```
%q	a single-quoted character literal safely escaped with Go syntax.
```

Reply:
It's the best I've found in this kind of circumstance (unknown strings).

Reply:
ok, let's go with it.